### PR TITLE
fix #46349 idtube.me

### LINKED ISSUE #46349: idtube.me
### Issue URL (Annoyance)
[https://idtube.me/film-seri/one-punch-man-2nd-season-2019-k0s6s1/playtv#](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fidtube.me%2Ffilm-seri%2Fone-punch-man-2nd-season-2019-k0s6s1%2Fplaytv%23)
### Comment
Username: @3xploiton3
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.3.4
Browser: | Firefox
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> ABPindo, <br/>AdBlockID

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#46349

### CODE REVIEW

File: EnglishFilter/sections/foreign.txt [hunk 1]
```diff
@@ -3208,6 +3208,10 @@ bhaskar.com#%#var canABP = true;
 !---------------------------------------------------------------------
 ! https://github.com/AdguardTeam/AdguardFilters/issues/46399
 sosok.grid.id#?#.container > h2.title__default > span:contains(/^PROMOTED CONTENT$/)
+! https://github.com/AdguardTeam/AdguardFilters/issues/46349
+103.194.171.75,idtube.me##.overlay.bm
```
Comment (Alex-302) [inline EnglishFilter/sections/foreign.txt]: cannot open the site. Is not it bookmark button?

Comment (Sergey-Lyapin) [reply EnglishFilter/sections/foreign.txt]: I was able to access this site with Singapore IP

Comment (Sergey-Lyapin) [reply EnglishFilter/sections/foreign.txt]: @Alex-302  I didn't find class .bm in any other elements
screenshot

Comment (Alex-302) [reply EnglishFilter/sections/foreign.txt]: This is not an ad. Also check another rules.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**